### ISSUE TITLE
Add Phoneinfoga Icon

### ISSUE THREAD

**Issue by hakrishi** [walkxcode/dashboard-icons#303]
### Type

Icon request

### Info

Ref:
https://sundowndev.github.io/phoneinfoga/
https://github.com/sundowndev/phoneinfoga

Thanks

### Icon(s)

![image](https://user-images.githubusercontent.com/59293085/222960168-02412cdc-8f54-4147-a704-8612b756753a.png)

**Comment by walkxcode** [comment]
All issues are temporarily on hold. Please refer to #316 for more info.

**Comment by walkxcode** [comment]
Hey, this icon has been added in the `new-standards` branch and will be merged once it is done!

https://github.com/walkxcode/dashboard-icons/blob/new-standards/png/phoneinfoga.png
https://github.com/walkxcode/dashboard-icons/blob/new-standards/png/phoneinfoga-light.png
https://github.com/walkxcode/dashboard-icons/blob/new-standards/svg/phoneinfoga.svg
